packages/template-block/package.json

```
@@ -0,0 +1,42 @@
+{
+    "name": "template-block",
+    "private": true,
+    "author": "Frontify Developers <developers@frontify.com>",
+    "version": "1.0.0",
+    "main": "dist/index.js",
+    "scripts": {
+        "serve": "frontify-cli serve --entryPath src/index.ts",
+        "deploy": "frontify-cli deploy --entryPath src/index.ts",
+        "lint": "eslint .",
+        "lint:fix": "eslint --fix .",
+        "prettier": "prettier --check .",
+        "prettier:fix": "prettier --write .",
+        "typecheck": "tsc --noEmit"
+    },
+    "devDependencies": {
+        "@babel/core": "^7.22.15",
```

```suggestion
        "@babel/core": "^7.22.17",
```

So we have all blocks on the same version